src/components/App.jsx

```diff
@@ -1,30 +1,17 @@
 import styles from '../styles/app'
 
 import React, { Component } from 'react'
-import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
 import { translate } from '../lib/I18n'
 import classNames from 'classnames'
 
-import { openFolder, alertClosed } from '../actions'
+import { alertClosed } from '../actions'
 
 import Alerter from 'cozy-ui/react/Alerter'
 import Sidebar from './Sidebar'
 import Topbar from './Topbar'
 
 class App extends Component {
-  componentWillMount () {
```

Comment:
One of the major changes: the App component is no longer the one who initializes data load. Since it's used across the whole app and the data that needs to be loaded changes on each view, I feel like this is the wrong place to do it.
Now the data is loaded by the appropriate containers, eg. `Folder` and `Trash`. This is also consistent with what we do in cozy-photos AFAIK.